account_manager/src/common.rs

```diff
@@ -1,6 +1,13 @@
+use account_utils::{read_mnemonic_from_user, strip_off_newlines};
 use clap::ArgMatches;
+use eth2_wallet::bip39::{Language, Mnemonic};
+use std::fs;
 use std::fs::create_dir_all;
 use std::path::{Path, PathBuf};
+use std::thread::sleep;
+use std::time::Duration;
+
+pub const MNEMONIC_PROMPT: &str = "Enter the 12-word mnemonic phrase:";
```

Comment:
We can parse mnemonics of varying length, the EF launchpad uses 24 words and we parse that. I think here we just say:

`Enter the 12-word mnemonic phrase:`